microsoft-edge/extensions-chromium/store-policies/developer-policies.md

```diff
@@ -159,218 +235,325 @@ If you publish a person's Personal Information to an outside service or third-pa
+
+Users should not be misled to click on advertisements to fully use the extension.
 
-#### 1.10.2 Policies and Agreements
+
+###### 1.10.2 Policies and agreements
 
 Any advertising content your extension displays must adhere to [Microsoft Advertising Network policies](https://about.ads.microsoft.com/policies/home).
 
 If your extension displays ads, all content displayed must conform to the advertising requirements of the [App Developer Agreement](/legal/windows/agreements/app-developer-agreement) and this Policy.
 
-#### 1.10.3 Quality of advertising
 
-*   The primary purpose of your extension must not be to get users to click ads.
+###### 1.10.3 Quality of advertising
+
+* The primary purpose of your extension must not be to get users to click ads.
 
-*   Your extension must not do anything that interferes with or diminishes the visibility, value, or quality of any ads that it does display.
+* Your extension must not do anything that interferes with or diminishes the visibility, value, or quality of any ads that it does display.
 
-#### 1.10.4 Promotions
+
+###### 1.10.4 Promotions
 
 If you purchase or create promotional ad campaigns to promote your extensions through the ad campaign functionality in [Partner Center](https://partner.microsoft.com/dashboard/microsoftedge/public/login?ref=dd), all ad materials you provide to Microsoft, including any associated landing pages, must comply with [Microsoft Creative Specifications Policy](https://about.ads.microsoft.com/policies/creative-specs) and [Microsoft Advertising Network policies](https://about.ads.microsoft.com/policies/home).
 
-#### 1.10.5 Notifying Users of Opt-Out for Interest-Based Advertising
+
+###### 1.10.5 Notifying users of opt-out for interest-based advertising
 
 Your privacy statement or terms of use must let users know you plan to send Personal Information to the ad service provider and must tell users how they may opt-out of interest-based advertising.
 
-#### 1.10.6 Other guidelines
+
+###### 1.10.6 Other guidelines
 
 If your extension is directed at children under the age of 13, as defined in the [Children's Online Privacy Protection Act](https://www.ftc.gov/tips-advice/business-center/privacy-and-security/children%27s-privacy); you must notify Microsoft of this fact in [Partner Center](https://partner.microsoft.com/dashboard/microsoftedge/public/login?ref=dd) and ensure that all ad content displayed in your extension is appropriate for children under the age of 13.
 
 
 <!-- ====================================================================== -->
-## 2 Content Policies
+## 2 Content policies
+
+The following policies apply to content and metadata (including publisher name, extension name, extension icon, extension description, extension screenshots, extension trailers and trailer thumbnails, and any other extension metadata) offered for distribution in the Microsoft Edge Add-ons store.
```

Comment:
remove "store"

Reply:
@Bhuvana-Priya - the sentence doesn't make sense without "store" at the end:
The following policies apply to content and metadata (including publisher name, extension name, extension icon, extension description, extension screenshots, extension trailers and trailer thumbnails, and any other extension metadata) offered for distribution in the Microsoft Edge Add-ons.

"for distribution in the Microsoft Edge Add-ons." wouldn't make sense, needs "store." at end

Reply:
If we link to the store, such that "Microsoft Edge Add-ons" becomes link text, omitting "store" or "website" might work better.

repo contains 23 hits in 12 files on the path:
https://microsoftedge.microsoft.com/addons/
![image](https://github.com/MicrosoftDocs/edge-developer/assets/45407486/290144f1-0d6c-4af1-a51a-208f147dbe96)


That short url redirects to:
https://microsoftedge.microsoft.com/addons/Microsoft-Edge-Extensions-Home - 13 hits in 7 files:
![image](https://github.com/MicrosoftDocs/edge-developer/assets/45407486/28ff3994-ae3a-4849-a42c-bd793e1d2d86)

Reply:
Changed to:
The following policies apply to content and metadata (including publisher name, extension name, extension icon, extension description, extension screenshots, extension trailers and trailer thumbnails, and any other extension metadata) offered for distribution in [Microsoft Edge Add-ons](https://microsoftedge.microsoft.com/addons/Microsoft-Edge-Extensions-Home).